src/Tools/dotnet-trace/CommandLine/Commands/CollectCommand.cs

```diff
@@ -255,55 +256,56 @@ private static string GetSize(long length)
         public static Command CollectCommand() =>
             new Command(
                 name: "collect",
-                description: "Collects a diagnostic trace from a currently running process",
-                symbols: new Option[] {
-                    CommonOptions.ProcessIdOption(),
-                    CircularBufferOption(),
-                    OutputPathOption(),
-                    ProvidersOption(),
-                    ProfileOption(),
-                    CommonOptions.FormatOption(),
-                    DurationOption()
-                },
-                handler: HandlerDescriptor.FromDelegate((CollectDelegate)Collect).GetCommandHandler());
+                description: "Collects a diagnostic trace from a currently running process") {
+                Handler = HandlerDescriptor.FromDelegate((CollectDelegate)Collect).GetCommandHandler()
+            }.AddOptions(new Option[] {
```

Comment:
If it's of interest, child `Command`, `Option`, and `Argument` instances can all be added to a `Command` using collection initializer syntax.

Reply:
@jonsequitur I don't see any method or property on Command that I could add a collection.

Reply:
I figured out collection initializers (works because Command is IEnumerable<> and has an Add(ISymbol) function) but it seems you can have both object (to set Handler) and collection initializers.

Reply:
True.